packages/uniorg-parse/src/parser.ts

```diff
@@ -1466,7 +1535,10 @@ class Parser {
     const contentsBegin = this.r.offset() + m.index + m[1].length + m[3].length;
     const contentsEnd = contentsBegin + m[4].length;
     this.r.resetOffset(contentsEnd + 1);
-    return u('code', { value }, []);
+    const begin = this.r.offset();
+    const end = begin + value.length;
```

Comment:
❗🐛 the `begin` here is set too late and points after `contentsEnd` (because of `resetOffset` on the previous line)

The correct `begin` should be set before `contentsBegin` to `this.r.offset() + m.index` (can use `this.r.advance(m.index)` right before it)